packages/ocaml-amqp/ocaml-amqp.0.0.1/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+maintainer: "Anders Fugmann <anders@fugmann.net>"
+authors: "Anders Fugmann <anders@fugmann.net>"
+homepage: "https://github.com/andersfugmann/ocaml-amqp"
+bug-reports: "https://github.com/andersfugmann/ocaml-amqp"
+license: "BSD"
+dev-repo: "git@github.com:andersfugmann/ocaml-amqp"
```

Comment:
OPAM is having difficulty with the dev-repo format.  Could you please change this line to

`dev-repo: "https://github.com/andersfugmann/ocaml-amqp.git"`

?